docs/devel/networking.md

```diff
@@ -326,3 +326,51 @@ Chain POSTROUTING (policy ACCEPT)
 target     prot opt source               destination
 MASQUERADE  all  --  10.0.2.2             anywhere
 ```
+
+### Masquerade binding using IPv6 addresses
+The masquerade binding mechanism is currently the only binding mechanism which
+accepts IPv6 addresses.
+
+It operates in exactly the same way as in IPv4, and follows the same goals:
+configure one to one NAT. As in it's IPv4 counter-part, the pods are reached
+via their IPv6 pod addresses.
+
+NAT is configured in the exact same way, but using ip6tables, or the `ipv6-nat`
+nftable table. Please refer to the tables below to visualize how NAT for IPv6
+addresses is accomplished in KubeVirt.
+
+```
+Chain PREROUTING (policy ACCEPT)
+target               prot opt source               destination
+KUBEVIRT_PREINBOUND  all      anywhere             anywhere
+
+Chain INPUT (policy ACCEPT)
+target     prot opt source               destination
+
+Chain OUTPUT (policy ACCEPT)
+target     prot opt source               destination
+DNAT       tcp      anywhere             localhost            tcp dpt:http to:fd10:0:2::2
+
+Chain POSTROUTING (policy ACCEPT)
+target     prot opt source               destination
+MASQUERADE  all      fd10:0:2::2          anywhere
```

Comment:
I would add a paragraph explaining the two most important rules here - MASQUERADE  and the DNAT.

Reply:
That has [already been explained for IPv4](https://github.com/kubevirt/kubevirt/commit/1d2295fcd9804db0f4965c3900d41934259bec84#diff-971521d7f1fea6eeaa462181948819a9R282-R286) and also [here](https://github.com/kubevirt/kubevirt/commit/1d2295fcd9804db0f4965c3900d41934259bec84#diff-971521d7f1fea6eeaa462181948819a9R318-R322); I don't think there's anything special for ipv6 - is it ?

Reply:
No, nothing spacial for ipv6.